packages/client/test/integration/miner.spec.ts

```diff
@@ -12,7 +17,7 @@ import type { EthereumClient } from '../../src/index.ts'
 
 async function setupDevnet(prefundAddress: Address) {
   const addr = prefundAddress.toString().slice(2)
-  const defaultChainData = {
+  const defaultChainData: Omit<GethGenesis, 'alloc'> = {
```

Comment:
Same nit here: should we default `alloc = {}` here instead of omitting it from the type?